### DISCUSSION TITLE
Feature Request: Volume Mixer - Remember position and size

### DISCUSSION THREAD

**Discussion by MBielefeld** [File-New-Project/EarTrumpet#1155] [Ideas]
I would find it very cool if the volume mixer would remember what position and size it had when it was closed and resume it the next time it was opened.
I have the mixer on a shortcut and it always opens in the background of games in full screen mode because it always opens in the same place on the primary monitor.

**Comment by TwoCables** [comment]
I'm a bit late to this, but you can use FancyZones in Microsoft PowerToys. I discovered this today and it works beautifully, once configured of course.
